faucet/templates/faucet/modal.html

```diff
@@ -10,11 +10,15 @@ <h5 class="modal-title">Help fund the faucet!</h5>
       <div class="modal-body">
         <p >
           Transparent donation address:</br>
-          <span class="taddress">tmKBPqa8qqKA7vrGq1AaXHSAr9vqa3GczzK</span>
+          <span class="taddress">tmP5cmDU66W2puHFoqfu8Zqe6M4mpsDk5fS</span>
```

Comment:
@charlieok do you know what addresses we should use for donations to the faucet?

Reply:
I am not sure... is this question still unresolved?